rskj-core/src/test/java/co/rsk/config/RskSystemPropertiesTest.java

```diff
@@ -223,4 +226,21 @@ void testGetRpcModulesWithObject() {
         assertEquals(9000, netModule.getTimeout());
         assertEquals(30000, netModule.getMethodTimeout("eth_getBlockByHash"));
     }
+
+    @Test
+    void testPeerLastSession() throws IOException {
```

Comment:
Woudn't be possible to load the actual configuration and check if the property it's enabled or not and if yes, check that the path to peersFile it's correctly configured? Yo

In my perspective, this should only validate the flags and not create and load the file, you could mock the File class. And the return from the filePath.

The test to generate the peer file and check if it was really generated could stay on the `DiscoveredPeerPersistenceServiceTest`  class.

Reply:
Here it is a simply test to check that we are reading the file provided by the config and reading it line by line. I have never mocked the file but it would be quicker than write and read one.

In relation with the enable property I could create another test for that.

Reply:
it's up to you, you could either create another test or refactor this one. :) 

In my point of view, I think that on this class would be good to validate that the flag and parameters necessary to write the file are working fine.